cases/drp-00-05.tex

```diff
@@ -26,32 +34,149 @@ \subsubsection{Intercase dependencies}
+  \end{enumerate}
+\end{enumerate}
+
+\paragraph{Execution}
+\begin{enumerate}
+  \item{The execution for each tract of the input data in \S\ref{sec:input} will be submitted to the hardware in \S\ref{sec:hardware} using the configuration in \S\ref{sec:configuration}.}
+  \item{During execution, the operator will use software to demonstrate the ability to check the processing status.}
+  \begin{enumerate}
+    \item{For LDM-503-2, the available Batch Production Service monitoring software consists of two commands: one to summarize currently submitted processing, one to get more details of single submission.}
+  \end{enumerate}
+  \item{If processing attempt completes, operator tags attempt as potential for release.  The campaign tags are used to make pre-release QA queries simpler.}
+  \item{If processing attempt fails, restart/resubmit on certain errors} \\
+    For LDM-503-2 test,
+    \begin{enumerate}
+        \item{Low level errors are automatically retried by framework (i.e., db connection fails, file transfer to/from data backbone fails).   For other errors, operator decides whether to just resubmit.   While a course-grained pipeline manual restart capability is available, the operator will simply resubmit the entire pipeline appropriately updating the state in the Oracle database.  Output files from previous attempts can be purged from the Data Backbone if no longer needed for debugging.}
+        \item{If a processing attempt is determined by the operator and production scientist to not be able to complete successfully, the attempt is not tagged and the problems are recorded.  All output files from a single failed processing attempt is kept in the Data Backbone for debugging purposes.} 
+    \end{enumerate}
+\end{enumerate}
+
+\paragraph{Non-science QA}
+\begin{enumerate}
+  \item{When the execution finishes, the success of the execution will be verified by checking the existence of the expected output data.
+  For each of the expected data products types (listed in \S\ref{drp-00-10-items})
+  and each of the expected units (PVIs, coadds, etc), the data product will be
+  verified to be non-empty.}
+    \begin{enumerate}
+        \item{Ask the Data Backbone about the files generated by each tract's execution through the pipeline.}
+        \item{Verify the physical and location information in Data Backbone DB matches the Data Backbone filesystem and vice-versa.} 
+    \end{enumerate}
+  \item{Check that for each data product type that the appropriate metadata was saved for each file as defined in \S\ref{sec:configuration}}
+  \item{Check provenance}
+    \begin{enumerate}
+        \item{Generically check that each file can be linked with the task and processing attempt that created it via the Data Backbone.}
+        \item{Generically check that the inputs for each step were saved to the Oracle database.}
+        \item{Specifically check conditions of DMS-REQ-0106.}
```

Comment:
What is "specifically" checked here? 0106 is itself quite vague ("a sufficient set of metadata attributes for users to re-create them in whole or in part") — are you going to enumerate the set of metadata which will be checked, or are you actually going to demonstrate the recreation of data products from metadata?

Reply:
I deleted this item as I think the two previous items are already sufficient. @MichelleGower please push back if you disagree